modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/AbstractDartCodegen.java

```diff
@@ -53,6 +54,7 @@ public abstract class AbstractDartCodegen extends DefaultCodegen {
     protected String pubAuthor = "Author";
     protected String pubAuthorEmail = "author@homepage";
     protected String pubHomepage = "homepage";
+    protected String pubRepository = "repository";
```

Comment:
shouldn't this by default be null ?

Reply:
TBH, I don't know what some of these fields are for :D 

I thought `"repository"` was for configuring some look up key, not to serve as a default value (as homepage is not a particularly good default value 🤔)

So you think it should be like this:
```suggestion
    protected String pubRepository = null;
```

Reply:
I think it should be null to retain backward compatibility, since pubspec.yaml is picky about the `repository` field https://pub.dev/documentation/pana/latest/models/Repository-class.html

Reply:
> TBH, I don't know what some of these fields are for :D

I have been contributing to this generator for ~2 years and I still get confused between them 😋

Reply:
Made it null